trajopt_optimizers/trajopt_optimizers_test_suite/CMakeLists.txt

```diff
@@ -0,0 +1,97 @@
+cmake_minimum_required(VERSION 3.5.0)
+project(trajopt_optimizers_test_suite)
+
+find_package(console_bridge REQUIRED)
+# find_package(Pagmo REQUIRED)
+find_package(OsqpEigen REQUIRED)
+find_package(trajopt_ifopt REQUIRED)
+find_package(trajopt_utils REQUIRED)
+# find_package(trajopt_pagmo REQUIRED)
+find_package(trajopt_sqp REQUIRED)
+find_package(ifopt REQUIRED)
+find_package(Eigen3 REQUIRED)
+find_package(GTest REQUIRED)
```

Comment:
Please remove commented packages and alphabetize. It makes it easier to manage when comparing packages.